### PR TITLE
chore: remove QuickSight explore user

### PR DESCRIPTION
**Pull request by tyyzqmf**
----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

## Summary

1. Remove QuickSight explore user
2. Create template dashboards in Publish user
3. Clean template dashboards when access analyses page 

## Implementation highlights

(describe how the merge request does for feature changes, share the RFC link if it has)

## Test checklist

- [ ] add new test cases
- [ ] all code changes are covered by unit tests
- [ ] end-to-end tests
  - [x] deploy web console with CloudFront + S3 + API gateway
  - [ ] deploy web console within VPC
  - [ ] deploy ingestion server
    - [ ] with MSK sink
    - [ ] with KDS sink
    - [ ] with S3 sink
  - [ ] deploy data processing
  - [ ] deploy data modeling
    - [ ] new Redshift Serverless
    - [ ] provisioned Redshift
    - [ ] Athena
  - [ ] deploy with reporting

## Is it a breaking change

- [ ] add parameters without default value in stack
- [ ] introduce new service permission in stack
- [ ] introduce new top level stack module

## Miscellaneous

- [ ] introduce new symbol link source file(s) to be shared among infra code, web console frontend, and web console backend

### CODE REVIEW

File: src/control-plane/backend/lambda/api/service/reporting.ts [hunk 1]
```diff
@@ -1030,11 +1019,7 @@ async function _cleanAnalyses(quickSight: QuickSight) {
 
   for (const analysisSummary of analysisSummaries) {
     const analysisId = analysisSummary.AnalysisId;
-    if (analysisSummary.Status !== ResourceStatus.DELETED
-      && analysisSummary.CreatedTime !== undefined
-      && analysisSummary.AnalysisId?.startsWith(QUICKSIGHT_TEMP_RESOURCE_NAME_PREFIX) &&
-      (new Date().getTime() - analysisSummary.CreatedTime.getTime()) > 60 * 60 * 1000
```
Comment (zxkane) [inline src/control-plane/backend/lambda/api/service/reporting.ts:1036]: Is it safe to remove the protection period if multiple users access the API?

**Review by zxkane — COMMENTED** [review]
Is there a way to remove the legacy explore user when upgrading the pipeline?

**Review by zxkane — COMMENTED** [review]
If upgrading from existing pipeline (without more projects with reporting created by < 1.1.3), could we remove the existing `QUICKSIGHT_EXPLORE_USER`?

**Comment by tyyzqmf** [comment]
> If upgrading from existing pipeline (without more projects with reporting created by < 1.1.3), could we remove the existing `QUICKSIGHT_EXPLORE_USER`?

delete explore user when all pipelines version higher than v1.1.3

**Review by zxkane — APPROVED** [review]

**Pull request merged**